nilearn/glm/contrasts.py

```diff
@@ -394,14 +448,23 @@ def compute_fixed_effects(contrast_imgs, variance_imgs, mask=None,
     fixed_fx_variance_img : Nifti1Image
         The fixed effects variance computed within the mask.
 
-    fixed_fx_t_img : Nifti1Image
-        The fixed effects t-test computed within the mask.
+    fixed_fx_stat_img : Nifti1Image
+        The fixed effects stat computed within the mask.
+
+    fixed_fx_z_score_img : Nifti1Image, optional
+        The fixed effects corresponding z-transform
+
+    Warns
+    -----
+    FutureWarning
+        Starting in version 0.13, return_z_score will default to True
```

Comment:
Clarification: would it not make more sense to say that starting from 0.13 will always a  Z score image? Because worded that way it implies we are planning to keep the `return_z_score` argument, which I do not think we should.

Reply:
```python
"""Before 0.13""" 
# do not return_z_score
(fixed_fx_contrast_img, 
 fixed_fx_variance_img,
 fixed_fx_stat_img) = compute_fixed_effects(
    contrast_imgs,
    variance_imgs,
    mask=None,
    precision_weighted=False,
    dofs=None,
    return_z_score=False,
)

# do return_z_score
(fixed_fx_contrast_img,
 fixed_fx_variance_img, 
 fixed_fx_stat_img,
 fixed_fx_z_score_img) = compute_fixed_effects(
    contrast_imgs,
    variance_imgs,
    mask=None,
    precision_weighted=False,
    dofs=None,
    return_z_score=True,
)

"""After 0.13"""
# always return_z_score
(fixed_fx_contrast_img,
 fixed_fx_variance_img, 
 fixed_fx_stat_img,
 fixed_fx_z_score_img) = compute_fixed_effects(
    contrast_imgs,
    variance_imgs,
    mask=None,
    precision_weighted=False,
    dofs=None
)
```